Documentation/docs.md

```diff
@@ -38,7 +37,20 @@ Administrators who need to create reliable and scalable key-value stores for the
  - [Disaster recovery][recovery]
  - [Performance][performance]
  - [Versioning][versioning]
- - [Supported platform][supported_platform]
+
+### Platform guides
+
+ - [Supported platforms][supported_platforms]
```

Comment:
this looks strange. we do not really include cloud provider in supported platform (since the platform here has a different definition: OS+ arch).

basically in this `Platform` section, we define `Platform` as two different things.